app/extensions/lineage_resolvers/endpoints/object_version.py

```diff
@@ -57,19 +57,14 @@ def _handler(
         event_dispatcher: EventDispatcher,
         object_uuid: UUID,
     ):
-        maybe_object: Optional[ObjectsTable] = object_repository.get_by_object_type_and_uuid(
+        object: Optional[ObjectsTable] = object_repository.get_by_object_type_and_uuid(
```

Comment:
`object` is een used type in python. Daarom zie je gebruik van objectx of in dit geval maar specified object_maybe. Zelfde is voor id en uuid, daar zie je dan steeds idx en uuidx